website/content/docs/commands/job/start.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: 'Commands: job start'
+description: |
+  The start command is used to start a stopped job.
+---
+
+# Command: job start
+
+The `job start` command is used to start a stopped job.
+
+The start command will use a Consul token with the following preference:
+first the `-consul-token` flag, then the `$CONSUL_HTTP_TOKEN` environment variable.
+Because the consul token used to [run] the targeted job version was not
+persisted, it must be provided to start if the targeted job version includes
+Consul Connect enabled services and the Nomad servers were configured to require
+[consul service identity] authentication.
+
+The Start command will use a Vault token with the following preference:
+first the `-vault-token` flag, then the `$VAULT_TOKEN` environment variable.
+Because the vault token used to [run] the targeted job version was not
+persisted, it must be provided to start if the targeted job includes
+Vault policies and the Nomad servers were configured to require [vault policy]
+authentication.
+
+## Usage
+
+```shell-session
+nomad job start [options] <job>
+```
+
+The `job start` command requires one input: the job ID. Nomad will start the
+latest version of the job that is not stopped. If the job has not previously
+been stopped, this command will fail.
+
+When ACLs are enabled, this command requires a token with the `submit-job`
+capability for the job's namespace. The `list-jobs` capability is required to
+run the command with a job prefix instead of the exact job ID. The `read-job`
+capability is required to monitor the resulting evaluation when `-detach` is
+not used.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Start Options
+
+- `-detach`: Return immediately instead of monitoring. A new evaluation ID
+  will be output, which can be used to examine the evaluation using the
+  [eval status] command.
```

Comment:
```suggestion
- `-detach`: Return immediately instead of monitoring. Nomad outputs a new
  evaluation ID, which you can use to examine the evaluation using the
  [eval status] command.
```